tests/Extensions/ComposerUpdateExtensionTest.php

```diff
@@ -2,10 +2,10 @@
 
 namespace BringYourOwnIdeas\UpdateChecker\Tests\Extensions;
 
+use BringYourOwnIdeas\Maintenance\Model\Package;
 use BringYourOwnIdeas\UpdateChecker\Extensions\ComposerUpdateExtension;
-use Package;
 use PHPUnit_Framework_TestCase;
-use SapphireTest;
```

Comment:
`protected $requiredExtensions` will need to become `protected static $required_extensions` in this file